rcache/gpusm compile error on v2.x

MTT build error on v2.x :
https://mtt.open-mpi.org/index.php?do_redir=2359

It seems includes are missing but the changes are so big it's not clear to me how this should be fixed.

`In file included from rcache_gpusm_module.c:41:0:
../../../../opal/mca/rcache/base/base.h:44:22: error: unknown type name ‘mca_base_framework_t’
 OPAL_DECLSPEC extern mca_base_framework_t opal_rcache_base_framework;`

--- a/opal/mca/rcache/base/base.h
+++ b/opal/mca/rcache/base/base.h
@@ -10,7 +10,7 @@
  *                         University of Stuttgart.  All rights reserved.
  * Copyright (c) 2004-2005 The Regents of the University of California.
  *                         All rights reserved.
- * Copyright (c) 2012-2015 Los Alamos National Security, LLC. All rights
+ * Copyright (c) 2012-2016 Los Alamos National Security, LLC. All rights
  *                         reserved.
  * $COPYRIGHT$
  *
@@ -27,7 +27,7 @@
 #include "opal_config.h"
 
 #include "opal/class/opal_list.h"
-#include "opal/mca/mca.h"
+#include "opal/mca/base/base.h"
 #include "opal/mca/rcache/rcache.h"
 
 BEGIN_C_DECLS

